contracts/src/deployers/ezeth/EzETHHyperdriveDeployerCoordinator.sol

```diff
@@ -138,11 +142,26 @@ contract EzETHHyperdriveDeployerCoordinator is HyperdriveDeployerCoordinator {
         IHyperdrive.PoolDeployConfig memory, // unused pool deploy config
         bytes memory // unused extra data
     ) internal view override returns (uint256) {
-        // Return ezETH's current vault share price.
+        return _convertToBase(ONE);
+    }
+
+    /// @dev Convert an amount of vault shares to an amount of base.
+    /// @param _shareAmount The vault shares amount.
+    /// @return baseAmount The base amount.
+    function _convertToBase(
+        uint256 _shareAmount
+    ) internal view returns (uint256) {
+        // Get the total TVL priced in ETH from restakeManager
         (, , uint256 totalTVL) = restakeManager.calculateTVLs();
-        uint256 ezETHSupply = ezETH.totalSupply();
 
-        // Price in ETH / ezETH, does not include eigenlayer points.
-        return totalTVL.divDown(ezETHSupply);
+        // Get the total supply of the ezETH token
+        uint256 totalSupply = ezETH.totalSupply();
+
+        return
+            renzoOracle.calculateRedeemAmount(
+                _shareAmount,
+                totalSupply,
+                totalTVL
+            );
     }
 }
```

Comment:
I don't think we need this to be in it's own function. You can just take the logic and plot it in `_getInitialVaultSharePrice`.

Reply:
roger, that was my initial method, but thought the extra clarity would be good for the audit.  I'll change to inline.